Modules/xASL_module_ASL.m

```diff
@@ -113,12 +113,6 @@
 %% D1. Load ASL parameters (inheritance principle)
 [~, x] = xASL_adm_LoadParms(x.P.Path_ASL4D_parms_mat, x, bO);
 
-if ~isfield(x,'Q')
```

Comment:
@MichaelStritt did you delete this because you already have this in this general field initialization/defaulting function that you made?

Reply:
Yes